earth_enterprise/src/common/timeutils.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2018 the Open GEE Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Basic time-related utilities
+
+#ifndef TIMEUTILS_H
+#define TIMEUTILS_H
+
+const double NSEC_TO_SEC = 1e-9;
+
+double getMonotonicTime();
```

Comment:
Kind of scary this is in the default namespace. Consider adding to some kind of namespace, maybe here and for the Profiler class too.